CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Add add-resin-supervisor-host-socket-service.service needed for web terminal. [Kostas]
```

Comment:
We usually don't add changelog entries for things that only affect the dind environment